terraform/files/bin/create_cluster.sh

```diff
@@ -157,7 +157,7 @@ echo "# Deploy services (CNI, OCCM, CSI, Metrics, Cert-Manager, Flux2, Ingress)"
 MTU_VALUE=$(yq eval '.MTU_VALUE' $CCCFG)
 if test "$USE_CILIUM" = "true" -o "${USE_CILIUM:0:1}" = "v"; then
   # FIXME: Do we need to allow overriding MTU here as well?
-  CILIUM_VERSION="v1.13.0"
+  CILIUM_VERSION="v1.13.3"
```

Comment:
```suggestion
  CILIUM_VERSION="v1.13.4"
```
why not **v1.13.4**?

Reply:
Ok I see, v1.13.4 seems to break IPsec